### ISSUE TITLE
Allow targeting of individual pastes (Feature Request)

### ISSUE THREAD

**Issue by Tumultuous-RGR-SiM** [anonymous-archiver/pastebin-dl#7]
To deal with the odd individual paste that was posted as a guest, or a single story paste that gets regular updates, I'll want to download just that one paste, without grabbing the other stuff in the author's bin. 

Can this be added as a feature?

**Comment by anonymous-archiver** [comment]
Yeah, shouldn't be too terrible to pull out the "download one file".  Biggest thing would be the whole "what if the file exists" situation that was discussed in the other ticket.
